### ISSUE TITLE
degraded image colors at https://frippery.org/busybox/

### ISSUE THREAD

**Issue by avih** [rmyorston/busybox-w32#289]
Looks like `BB_SKIP_ANSI_EMULATION=0` at the screenshot https://frippery.org/busybox/busybox.png , which arguably looks worse than with 256 or true colors.

**Comment by rmyorston** [comment]
No, it's a Windows 11 legacy mode console. I wasn't prepared for the radical new look of the terminal.

**Comment by avih** [comment]
> No, it's a Windows 11 legacy mode console

I don't have Windows 11 readily. Does that mean that by default the console in Windows 11 does not support switching on the VT mode?

**Comment by rmyorston** [comment]
The Windows 11 _legacy_ console doesn't support VT mode, but the same is true in Windows 10.

In Windows 10 the legacy and non-legacy (modern?) console look the same. In Windows 11 they look different.

Next time I take a screenshot I might just go back to Windows 10. Or I might try tweaking the appearance of the modern Windows 11 console. (Though, in general, I think life's too short to waste time fiddling with that sort of thing.)

**Comment by avih** [comment]
FWIW, I'm busybox-w32 (and w64devkit) with the [Windows terminal](https://github.com/microsoft/terminal) on Windows 10, and I quite like it.

**Comment by rmyorston** [comment]
I had to reinstall my Windows 11 VM the other day and was surprised to find that the standard console was able to display true colour. I was later further surprised to find that starting a BusyBox session brought up a new-style terminal instead of the traditional console.

It appears Microsoft had decided (without my approval) to change the setting of the terminal so it would be used whenever a console was required. It's all very confusing.

Anyway, now that I'm wise to their tricks I've been able to revert to a colourful traditional console. The web page has been updated with a new screenshot which does justice to the wonders of `shlolcat`.
